### PR TITLE
re #889 Update Naming

### PR DESCRIPTION
**Pull request by kahboom**
cc @EricWittmann - Still working on the last 7 blog posts.

### CODE REVIEW

**Comment by EricWittmann** [comment]
OK cool - thanks @kahboom  - should I wait to merge until those last 7 are done?

**Comment by kahboom** [comment]
@EricWittmann - Up to you, I'm guess it will take me about an hour or two to look over everything, make sure it looks okay, and maybe have time to edit the images. Right now I only need about 4 more posts (no images).

**Comment by EricWittmann** [comment]
OK great - I'll wait until you've got the rest ready I guess. :)

File: _blog-src/_layouts/post.html [hunk 1]
```diff
@@ -11,6 +11,21 @@ <h1 style="color: #666; font-weight: bold" class="post-title section-header">{{
...
+  {% if page.newUrl || page.oldUrl %}
+  <div class="name-change-alert" style="margin-top:15px;">
+    {% if page.newUrl %}
+    <!-- Old Post -->
+    <p>NOTE: A newer revision of this blogpost correct for <a href="{{ site.url }}{{ site.baseurl }}/{{ page.newUrl }}">apiman 1.2.x is available</a>. This version written for apiman 1.1.x, for which it remains valid.</p>
+    {% endif %}
+
+    {% if page.oldUrl %}
+    <!-- New Post -->
+    <p>NOTE: This is a redux of our popular Keycloak OAuth2 for *apiman 1.2.x*. If you're still using apiman 1.1.x, you can refer to the <a href="{{ site.url }}{{ site.baseurl }}/{{ page.oldUrl }}">older revision</a>.</p>
```
Comment (msavy) [inline _blog-src/_layouts/post.html]: > NOTE: This is a redux of our popular Keycloak OAuth2 

I guess this is artefact when you copied over the 'old/new blog post' notices from the KC blogpost. Presume you'll want to make it more generic. In asciidoctor `NOTE:` is an admonition, and it doesn't work in markdown, so suggest you kill that bit.

Comment (kahboom) [inline _blog-src/_layouts/post.html]: haha, good catch. I'm sure that'd confuse quite a few people.

Comment (kahboom) [inline _blog-src/_layouts/post.html]: @msavy - Maybe something like this:

> Note: This is a redux of an existing post for apiman 1.2.x. If you're still using apiman 1.1.x, you can refer to the older revision.

Comment (msavy) [inline _blog-src/_layouts/post.html]: How’s about:

This is a redux of an existing blogpost for apiman 1.2.x […]

Are you redoing all of them or just popular ones? If only popular ones I vote s/an existing/a popular/

Comment (kahboom) [inline _blog-src/_layouts/post.html]: @msavy - All of them since there were only about 13 of them.

**Comment by kahboom** [comment]
@EricWittmann - I pushed up my changes yesterday, which covers all blog posts, however, I'd like for you to take a look at the `2015-05-19-rest-api-redux.markdown` post to make sure everything was updated properly, as I felt the least confident with those changes. I'm just editing some images for the blog posts now and will do a sweep through the blog in case I broke anything.

**Comment by kahboom** [comment]
screen shot 2016-01-12 at 9 44 50 am

Fixed the styling, updated the text (see screenshot), and fixed an issue where the newer posts were not displaying the notice block. Working on images and also, I'm not sure what we want to do about duplicate entries in the list of blog posts, if anything at all, as that may be confusing for users.

**Pull request merged**